### ISSUE TITLE
Path Mapping is not properly work

### ISSUE THREAD

**Issue by berlysia** [ReactiveX/rxjs#3330]
**RxJS version:**
6.0.0-alpha.3

**Code to reproduce:**
```js
import { of } from 'rxjs';
import { map } from 'rxjs/operators';
import { ajax } from 'rxjs/ajax';
import { websocket } from 'rxjs/websocket';
import { TestScheduler } from 'rxjs/testing';
```

with webpack configuration for `_esm5` described in [doc/pipeable-operators.md#build-and-treeshaking](https://github.com/ReactiveX/rxjs/blob/ddffecc32617c68df793f14b5e4a5f49af8257f2/doc/pipeable-operators.md#build-and-treeshaking)

FYI: reproduced in webpack 3.11.0 & 4.0.0-beta.2

**Expected behavior:**
All import paths are mapped into `_esm5` directory.

**Actual behavior:**
All import paths are mapped into root directory, not in `_esm5`.

**Additional information:**
```js
import { of } from 'rxjs/index';
import { map } from 'rxjs/operators/index';
import { ajax } from 'rxjs/ajax/index';
import { websocket } from 'rxjs/websocket/index';
import { TestScheduler } from 'rxjs/testing/index';
```

works as expected.

**Comment by benlesh** [comment]
cc @jasonaden

**Comment by BioPhoton** [comment]
Hi @berlysia !

As this is quite a while ago, is it probably already solved? 

If yes please close that one. Otherwise please give a quick update of the current status.

**Comment by berlysia** [comment]
This issue is already solved in v6.0.0 and also in v6.5.3. :sparkles:
